CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Circular/linear polarizations are supported in polarization dropdowns for saving subimages and generating hypercubes.
 * The ability to export high resolution png images for publication quality in journals.
 * The ability to use a custom rest freqency for spectral matching, spectral axis display, and PV image x/y axis display.
+* Added PV generator (static).
```

Comment:
I'd like a little more detail here. Something like:

`Added new feature: ability to generate a position-velocity (PV) image from a line region on images with a supported coordinate system. The generated images are loaded as separate images, similar to generated moment maps.`

Reply:
has changed.